packages/fluentui/react-northstar/src/components/Popup/Popup.tsx

```diff
@@ -585,7 +585,7 @@ Popup.propTypes = {
   positionFixed: PropTypes.bool,
   renderContent: PropTypes.func,
   target: PropTypes.any,
-  trigger: customPropTypes.every([customPropTypes.disallow(['children']), PropTypes.any]),
+  trigger: customPropTypes.shorthandAllowingChildren,
```

Comment:
Changed so the example from `PopupCustomTargetExample` doesn't complain about conflict of props

Reply:
Please revert this and update an example as in: https://github.com/microsoft/fluentui/pull/12759

Reply:
Done